src/main/java/Storage.java

```diff
@@ -0,0 +1,81 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+
+public class Storage {
+    private String fileLocation =  "data/duke.txt";
```

Comment:
Try not to hard code this to be the text file. maybe it could be the main's choice instead so different instances can have different file paths